src/content/developers/tutorials/Using-UseDApp-with-unity/index.md

```diff
@@ -0,0 +1,206 @@
+---
+title: Using UseDApp with Unity
+description: Learn how to use UseDApp to interact with Unity engine project 
+author: "lucasespinosa28"
+tags: ["usedapp", "unity","react","dapp"]
+skill: intermediate
+lang: en
+sidebar: true
+sourceUrl: https://github.com/lucasespinosa28/usedappwithUnity
+---
+## Introduction {#introduction}
+In this tutorial, you will learn how to use [UseDApp](https://usedapp.readthedocs.io/en/latest/) a React plugins for rapid Dapp development, 
+and how to interact with a Unity using react, you will get your account balance inside a Unity and send a transaction from unity.
+
+### You need have [Unity](https://unity.com/) installed 
```

Comment:
Would make this just a bold statement wrapped in double-asterisks instead of an empty header section